cogs/polling.py

```diff
@@ -0,0 +1,56 @@
+import discord
+import aiohttp
+import asyncio
+from urllib.request import urlopen
+import json, time
+
+from discord.ext import commands
+from .utils import checks
+
+
+class CommitTracker:
+    def __init__(self, bot):
+        self.bot = bot
+        self.base_commit = bot.config['basecommit']
+        self.wait_time = 120
+
+    async def get_latest_commit(self, owner, repo):
+        url = 'https://api.github.com/repos/{owner}/{repo}/commits?per_page=1'.format(owner=owner, repo=repo)
+        async with aiohttp.ClientSession() as session:
+            async with session.get(url) as response:
+                return await response.text()
+
+    async def trackCommits(self):
+        await self.bot.wait_until_ready()
+        commit = self.bot.config['basecommit']
+        while True:
+            oldcommit=commit
+            owner = 'kwsch'
+            repo = 'PKHeX'
+            data = await self.get_latest_commit(owner, repo) 
+            try:
+                commitdata = json.loads(data)[0]
+            except KeyError:
```

Comment:
Is there any circumstance under which KeyError will occur?

Reply:
Happened a few times, it keeps executing but randomly throws a `KeyError` at index 0 exception. I managed to get the Json printed out in one instance and it said the following 
```json
{"message" : "Server Failure"}
```
This doesn't affect the execution as the code still keeps looping, but it does happen once in a while